AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -2066,6 +2066,7 @@ besplatka.ua##.footer_bottom > div#applications
 klook.com,newchic.com,m.zaful.com,c.mi.com,m.vseinstrumenti.ru##.app-download
 m.avito.ru##.promo-app-inner
 m.17track.net##.yq-panel-download
+m.17track.net##div[class^="ProductDetails_appBox__"]
```

Comment:
Wrong section - onmy mobile version.

https://marketplace.visualstudio.com/items?itemName=ExodiusStudios.comment-anchors this extension shows more detailed navigation. Only v1.9.6